package.json

```diff
@@ -27,6 +27,7 @@
     "is-ci": "^3.0.0",
     "is-windows": "^1.0.2",
     "js-yaml": "4.1.0",
+    "listr": "^0.14.3",
```

Comment:
I don't remember why we stopped using listr. Doesn't look like its maintained anymore for one. 

@dan-barrett can you remember why we switched to cli-ux?

Reply:
Nope, I don't remember. I don't think I added that. But my guess would be that we did it to kee using [oclif](https://github.com/oclif/cli-ux) wherever we can.

Reply:
I see that it hasn't been worked on for a while, but I used if because oclif still recommends it at least for spinners. Assumed the whole thing was fair game

https://oclif.io/docs/spinner

Reply:
Looks like listr is pretty dead - I noticed there is a listr2 with some traction:
https://www.npmjs.com/package/listr2